### PR TITLE
Feat/add byron example

### PR DESCRIPTION
**Pull request by MMortaraAtix**
# Description

_A clear and concise description of what this pull request does or fixes._

# Proposed Solution

_**Optional** Explain how does this PR solves the problem stated in [Description](#Description). You can also enumerate different alternatives considered while approaching this task._

# Important Changes Introduced

_**Optional** Notice Reviewers about changes that were introduced while developing this task_

# Testing

_**Optional** Leave some recommendations should be useful while reviewers are testing this PR_

### CODE REVIEW

**Review by jvieiro — APPROVED** [review]

File: yarn.lock [hunk 1]
```diff
@@ -0,0 +1,4 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```
Comment (tomasBustamante) [inline yarn.lock:1]: Is this file necessary here? :thinking:
